src/StorageMover/StorageMover.Autorest/custom/NewAzStorageMoverMultiCloudConnectorEndpoint.ps1

```diff
@@ -149,9 +149,6 @@ function New-AzStorageMoverMultiCloudConnectorEndpoint {
         $Properties.EndpointType = "AzureMultiCloudConnector"
         $PSBoundParameters.Add("Property", $Properties)
 
```

Comment:
Is the default value returned from server "SystemAssigned"? If not this could be a possible breaking chang

Reply:
Actually the original PR for this new api version is not released yet, this is just a fix PR in succession to that.:
https://github.com/Azure/azure-powershell/pull/28488

So should not cause any issue.